datanode/gateway/graphql/schema.graphql

```diff
@@ -4912,6 +4912,36 @@ type LiquidityProvision {
   status: LiquidityProvisionStatus!
   "A reference for the orders created out of this liquidity provision"
   reference: String
+  "Liquidity provision that has been updated by the liquidity provider, and has been accepted by the network, but will not be active until the next epoch."
+  pending: PendingLiquidityProvision
+}
+
+"Liquidity provision that has been accepted by the network, but is not yet active."
+type PendingLiquidityProvision {
+  "Unique identifier for the order (set by the system after consensus)"
+  id: ID!
+  "The party making this commitment"
+  party: Party!
+  "RFC3339Nano time when the liquidity provision was initially created"
+  createdAt: Timestamp!
+  "RFC3339Nano time of when the liquidity provision was updated"
+  updatedAt: Timestamp
+  "Market for the order"
+  market: Market!
+  "Specified as a unit-less number that represents the amount of settlement asset of the market."
+  commitmentAmount: String!
+  "Nominated liquidity fee factor, which is an input to the calculation of liquidity fees on the market, as per setting fees and rewarding liquidity providers."
+  fee: String!
+  "A set of liquidity sell orders to meet the liquidity provision obligation."
+  sells: [LiquidityOrderReference!]!
+  "A set of liquidity buy orders to meet the liquidity provision obligation."
+  buys: [LiquidityOrderReference!]!
+  "The version of this liquidity provision"
+  version: String!
+  "The current status of this liquidity provision"
+  status: LiquidityProvisionStatus!
+  "A reference for the orders created out of this liquidity provision"
```

Comment:
```suggestion
  "A reference for the orders created to support this liquidity provision"
```
I think this is also a holdover from the shapes era, so it's worth checking these order ones are the same functionality